tests/baseline/test_baseline.py

```diff
@@ -24,20 +24,32 @@
 
 LOG = logging.getLogger("test_baseline")
 
+EXCLUDED_ATTRIBUTES = [
```

Comment:
This looks like it might be a remnant of the past, as in the end `content` is just a plain method and not an `attr.ib` - it should already not appear in the output of `asdict` and therefore not need any filtering.